src/services/impl/StringTextWriter.hh

```diff
@@ -0,0 +1,47 @@
+#ifndef RA_SERVICES_STRINGTEXTWRITER
+#define RA_SERVICES_STRINGTEXTWRITER
+#pragma once
+
+#include "services\TextWriter.hh"
+
+#include "RA_StringUtils.h"
+
+namespace ra {
+namespace services {
+namespace impl {
+
+class StringTextWriter : public ra::services::TextWriter
+{
+public:
+    explicit StringTextWriter() noexcept
```

Comment:
You should probably get rid of the  explicit here. It's only useful with default constructors with one or more parameters. Since most people here like assignment instead of initializers itll be too verbose as it is.

Reply:
fixed